### DISCUSSION TITLE
jet fans

### DISCUSSION THREAD

**Discussion by bkhaksari** [firemodels/fds#12758] [General]
Hi all,

I use a one-dimensional model for longitudinal airflow inside the tunnel and then use its results as boundary conditions in FDS.

I was wondering if I want to add jet fans to FDS as well, what velocity profile do I need to use for them? Should I use the same boundary condition velocity profile for jet fans too?

Many thanks,

**Comment by Er9y714** [comment]
If you are interested in how to model Jet fans you might have a look at [this post](https://support.thunderheadeng.com/tutorials/pyrosim/modeling-jet-fans/).
